### ISSUE TITLE
Nixvim fails to build because of manpages generation error

### ISSUE THREAD

**Issue by ivandimitrov8080** [nix-community/nixvim#754]
**Nixpkgs channel**: nixpkgs

## Description
```
error: builder for '/nix/store/2304z03s4synvazxcvp04kw8v018xpd1-nixvim-configuration-reference-manpage.drv' failed with exit code 1;
       last 10 log lines:
       >   File "/nix/store/fapl15ayqmrgcykflykjz7xwkzlrhj1b-nixvim-render-docs-0.0/lib/python3.11/site-packages/nixvim_render_docs/__init__.py", line 85, in run_options
       >     run_manpage_options(args)
       >   File "/nix/store/fapl15ayqmrgcykflykjz7xwkzlrhj1b-nixvim-render-docs-0.0/lib/python3.11/site-packages/nixvim_render_docs/__init__.py", line 75, in run_manpage_options
       >     md = NixvimManpageConverter(revision=args.revision)
       >          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       >   File "/nix/store/fapl15ayqmrgcykflykjz7xwkzlrhj1b-nixvim-render-docs-0.0/lib/python3.11/site-packages/nixvim_render_docs/__init__.py", line 18, in __init__
       >     super().__init__(revision, _options_by_id=_options_by_id)
       > TypeError: ManpageConverter.__init__() missing 2 required positional arguments: 'header' and 'footer'
       > error:
       > ManpageConverter.__init__() missing 2 required positional arguments: 'header' and 'footer'
```

Whenever I try to build.

## Config

```nix
  defaultConfig = rec {
    globals = {
      mapleader = " ";
      maplocalleader = " ";
    };
    options = {
      number = true;
      scrolloff = 15;
      hlsearch = false;
      updatetime = 20;
      autoread = true;
      tabstop = 4;
      shiftwidth = 2;
      expandtab = true;
    };
    colorschemes.catppuccin = {
      enable = true;
      flavour = "mocha";
      transparentBackground = true;
      integrations = {
        cmp = true;
        gitsigns = true;
        treesitter = true;
        telescope = {
          enabled = true;
        };
        markdown = true;
      };
    };
    keymaps = [
      {
        mode = [ "n" "v" ];
        key = "<Space>";
        action = "<Nop>";
        options.silent = true;
      }
      {
        mode = "n";
        key = "<Tab>";
        action = "<cmd>BufferNext<cr>";
        options.silent = true;
      }
      {
        mode = "n";
        key = "<S-Tab>";
        action = "<cmd>BufferPrevious<cr>";
        options.silent = true;
      }
      {
        mode = "n";
        key = "<leader>x";
        action = "<cmd>BufferClose<cr>";
        options.silent = true;
      }
      {
        mode = "n";
        key = "<leader>/";
        action = "require('Comment.api').toggle.linewise.current";
        lua = true;
        options.silent = true;
      }
      {
        mode = "v";
        key = "<leader>/";
        action = "<ESC><cmd>lua require('Comment.api').toggle.linewise(vim.fn.visualmode())<cr>";
        options.silent = true;
      }
      {
        mode = [ "n" "t" ];
        key = "<leader>h";
        action = "<cmd>ToggleTerm<cr>";
        options.silent = true;
      }
      {
        mode = "n";
        key = "<leader>ff";
        action = "require('telescope.builtin').find_files";
        lua = true;
      }
      {
        mode = "n";
        key = "<leader>fw";
        action = "require('telescope.builtin').live_grep";
        lua = true;
      }
      {
        mode = "n";
        key = "<leader>e";
        action = "vim.diagnostic.open_float";
        lua = true;
      }
      {
        mode = "n";
        key = "<leader>ca";
        action = "vim.lsp.buf.code_action";
        lua = true;
        options.silent = true;
      }
      {
        mode = "n";
        key = "<leader>lr";
        action = "vim.lsp.buf.rename";
        lua = true;
        options.silent = true;
      }
      {
        mode = "n";
        key = "<leader>lf";
        action = "vim.lsp.buf.format";
        lua = true;
        options.silent = true;
      }
    ];
    plugins = {
      cmp-spell.enable = true;
      cmp-nvim-lsp.enable = true;
      cmp_luasnip.enable = true;
      barbar.enable = true;
      luasnip.enable = true;
      telescope.enable = true;
      gitsigns.enable = true;
      toggleterm.enable = true;
      comment-nvim.enable = true;
      treesitter.enable = true;
      ts-autotag.enable = true;
      nvim-autopairs.enable = true;
      lsp = {
        enable = true;
        servers = {
          nixd = {
            enable = true;
            settings.formatting.command = "nixpkgs-fmt";
          };
          efm = {
            extraOptions = {
              init_options = { documentFormatting = true; };
            };
          };
        };
        onAttach = ''
          if client.server_capabilities.documentHighlightProvider then
              vim.api.nvim_create_autocmd("CursorHold", {
                  buffer = bufnr,
                  callback = function()
                      vim.lsp.buf.document_highlight()
                  end,
              })
              vim.api.nvim_create_autocmd("CursorMoved", {
                  buffer = bufnr,
                  callback = function()
                      vim.lsp.buf.clear_references()
                  end,
              })
          end
        '';
        capabilities = ''
          require("cmp_nvim_lsp").default_capabilities()
        '';
        keymaps = {
          lspBuf = {
            K = "hover";
            gD = "references";
            gd = "definition";
            gi = "implementation";
            gt = "type_definition";
          };
        };
      };
      efmls-configs = {
        setup = {
          typescript.formatter = "prettier_d";
          typescriptreact.formatter = "prettier_d";
          html.formatter = "prettier_d";
          css.formatter = "prettier_d";
          json.formatter = "prettier_d";
        };
      };
      nvim-cmp = {
        enable = true;
        mapping = {
          "<CR>" = "cmp.mapping.confirm({ select = true })";
          "<C-Space>" = "cmp.mapping.complete()";
          "<C-j>" = "cmp.mapping.select_next_item()";
          "<C-k>" = "cmp.mapping.select_prev_item()";
        };
        snippet.expand = "luasnip";
      };
      nvim-cmp.sources = [
        { name = "nvim_lsp"; }
        { name = "luasnip"; }
        { name = "path"; }
        { name = "buffer"; }
      ];
      lualine = {
        enable = true;
        theme = "catppuccin";
      };
    };
  };
```

**Comment by ivandimitrov8080** [comment]
Workaround - set:

```nix
nixvim.enableMan = false;
```

**Comment by siph** [comment]
I think they're just waiting on nixpkgs: #751

**Comment by Iggle** [comment]
It's a bit unfortunate that home-manager development tracks nixpkgs-unstable and nixvim development nixos-unstable. Most people not on a stable home-manager channel will probably use nixpkgs-unstable.

**Comment by traxys** [comment]
We could maybe follow what home-manager does and track nixpkgs-unstable

**Comment by GaetanLepage** [comment]
Honestly, I don't really care.
It should not be too important as both channels are often not spaced by more than 3-4 days.

Actually, we only bump our flake inputs once a week and we are thus always lagging by a few days behind the current state of the channel we are following.

An important thing to know is that whatever nixpkgs revision is currently pinned in the nixvim flake's inputs **doesn't matter** if you are using it as a HM/NixOS module.
When using the HM or NixOS nixvim module, nixvim will use whatever nixpkgs HM/NixOS gives it.
Thus, if a bug lands in the nixpkgs-unstable (or nixos-unstable) channel and that you update your inputs before we had time to find out about the problem and fix it, you will get a broken system.

The channel we target is supposed to mean "if a bug is present on this channel, we will fix it ASAP".

**Comment by GaetanLepage** [comment]
Fixed by #760
